app.py

```diff
@@ -52,13 +59,28 @@ def scrape_schedules():
         logging.info("Scraping game schedules...")
         fetch_game_schedule()
 
-    @scheduler.task("interval", id="scrape_schedules", seconds=43200)
+    @scheduler.task("interval", id="scrape_schedules", seconds=43200) # 12 hours
     def scrape_videos():
         logging.info("Scraping YouTube videos...")
         fetch_videos()
 
     scrape_schedules()
     scrape_videos()
 
+if not args.no_daily_sun:
+    @scheduler.task("interval", id="scrape_daily_sun", seconds=3600)
+    def scrape_daily_sun():
+        logging.info("Getting Daily Sun Sports News...")
+        fetch_news()
+
+    @scheduler.task("interval", id="cleanse_daily_sun_db", seconds=604800) # 1 week
+    def cleanse_daily_sun_db():
+        logging.info("Cleaning the Daily Sun database from old articles...")
+        ArticleService.cleanse_old_articles()
+
```

Comment:
not entirely sure but there might be a period where we get all article and then immediately wipe them (after the week interval), leading to an hour of no articles in our database

Reply:
I only delete stuff older than a week

Reply:
oh my b i missed that, no worries then